build.gradle

```diff
@@ -17,6 +17,8 @@ dependencies {
 	implementation  'org.springframework.boot:spring-boot-starter-web'
 	implementation ('org.springframework.boot:spring-boot-starter-data-jpa')
 	implementation('org.postgresql:postgresql')
+//	implementation 'org.springframework.boot:spring-boot-starter-security'
```

Comment:
Normally it should be removed, but next task is related with security so you can leave it like that.